test/Robot_Localization_Testcases.sh

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Would you mind renaming these files so that they are all lower-case? `robot_localization_test_cases.sh` is more consistent with the rest of the package, as long as it doesn't violate some new ROS2 standard that I'm not aware of.

Same goes for all the other new test files with capital letters in the name.

Reply:
Renamed file names in lower-case... https://github.com/cra-ros-pkg/robot_localization/commit/862d447ac9afc65d694246c76082368d65cf9f25